src/UI/Component/Toast/Factory.php

```diff
@@ -40,4 +38,31 @@ interface Factory
      * @return  \ILIAS\UI\Component\Toast\Toast
      */
     public function standard($title, Icon $icon) : Toast;
+
+    /**
+     * ---
+     * description:
+     *   purpose:
+     *     Toasts Containers displays Toasts in the top right corner of the ILIAS page content.
```

Comment:
```suggestion
     *     Toasts Containers display Toasts on the ILIAS page.
```
It's often simpler if we do not reference the locations on the page. The "top right corner" will already be false on small screens...